### ISSUE TITLE
Is this project alive, there seems to be no update?

### ISSUE THREAD

**Issue by muhajirdev** [creativetimofficial/material-kit-vuejs#1]

**Comment by awakenedguy** [comment]
Would be really cool to see it alive for sure. I would buy a PRO version, if it were available.

**Comment by lukeramsden** [comment]
Oh Jesus, completely forgot about this. Life has been so busy this year, new job, new place, the lot.

If there really is interest, I could probably start on it next week properly as the projects I've had over the last year are wrapping up.

**Comment by awakenedguy** [comment]
Material Kit Pro is Creative Tim's the most bought paid template, as far as I can see it from outside (without being Creative Tim's employee). Unhappily it only exist as a "raw" html, JQuery-based version. No Vue or React version exist, which is surely a shame, as the demand for Vue/React versions should normally be higher than for their raw html/css/jquery counterparts. Especially for Vue. At least IMHO.

**Comment by lukeramsden** [comment]
Yeah, that makes sense to me. I'll try and start a plan this weekend 😄

**Comment by alexandru-paduraru** [comment]
@awakenedguy and @lukeramsden haha, sincerely I completely forgot about this too. We also have a React version which is WIP: https://github.com/creativetimofficial/material-kit-react. We are doing the Bootstrap 4 conversion of Material Kit Free and PRO and then we will release the React and Vue versions. Yes, there is a high demand in Vue and React and we want to cover that too but it's a lot of work with the current Bootstrap 3 to Bootstrap 4 products (we have 40 products at this moment). Hopefully, we will have some releases for React and Vue in a couple of weeks.

We also believe VueJS community will grow a lot and we sponsor it: https://opencollective.com/creative-tim

Any help will be highly appreciated :D

Best,
Alex

**Comment by lukeramsden** [comment]
@alexandru-paduraru Well now my time is freeing up, I can go all-in on helping, so now it's up to you: shall I wait until the B4 conversion is done? Please let me know what you want me to do.

**Comment by minikN** [comment]
@lukeramsden @alexandru-paduraru Are you still going to use [Bootstrap Material Design](https://fezvrasta.github.io/bootstrap-material-design/)? If so, you'd still need jQuery. You could use
[vuematerial](http://vuematerial.io/#/) (at least for Vue) instead. I'd love to see this live!

**Comment by alexandru-paduraru** [comment]
@minikN and @lukeramsden, sincerely don't know which version people would like more. There is also this one: https://bootstrap-vue.js.org/ which can be used as starting point. So at this moment I see 2 options:

1. Bootstrap Vue + Bootstrap 4 Material Design + our own Material style and pages
2. Vue Material + our own Material style and pages, but then we will miss some elements from Bootstrap

What do you think?

**Comment by lukeramsden** [comment]
The current repo uses neither, just uses bootstrap CSS. But the first one seems like the best idea.

**Comment by minikN** [comment]
@lukeramsden @alexandru-paduraru But if you are going to use Bootstrap Material Design, you are also going to use jQuery. There is little point in using vue if you're also going to use jQuery. Am I wrong?
I'm don't want to dictate how you should build this project, but in my opinion, if building a vue-project, one should avoid using jQuery if possible (and in this case it is possible). Vuematerial offers a fully integrated grid system and tons of components, just look at a couple of them and how many external jQuery plugins you could save:

[Snackbars](http://vuematerial.io/#/components/snackbar) - no more Bootstrap Notify
[Wizard](http://vuematerial.io/#/components/stepper) (With Validation) - You could even include your Material Wizard quite easily
[Tooltips](http://vuematerial.io/#/components/tooltip), [Mobile Navigation](http://vuematerial.io/#/components/sidenav), Datepicker coming in next release.. and just a tone more.

You can read all about the upcoming release [here](https://github.com/vuematerial/vue-material/issues/1081). I know, Im heavily advocating the project. I'm not part of it whatsoever, I just think it's a way better option then falling back to using a decade old, slow library like jQuery.

Cheers.

**Comment by alexandru-paduraru** [comment]
@minikN yes, you're right, there would be some dependencies on jQuery. So probably the solution would be to start with Vue Material. Can you please put us in contact with the creator? We would like to address some questions before starting the actual development.

Best,
Alex

**Comment by minikN** [comment]
@alexandru-paduraru Done sir. I informed him about your plans to use his project. You can also contact him directly: https://github.com/marcosmoura
Hope to see this live on day!
@lukeramsden In case there is need, I'm happy to contribute.

**Comment by marcosmoura** [comment]
Hi There @alexandru-paduraru !
I'm the creator of Vue Material. Glad to see the interest in integrate my project.
How can I help?

**Comment by alexandru-paduraru** [comment]
@marcosmoura hi mate! Congrats for building such a great product! I will come back these days with some questions then we will start the development.

@minikN and @lukeramsden, since we want to use this product as the base for all our future Material products I will talk with Marco and my team and then we will see which is the timeline for doing things. If you want to contribute it would be great. We will release the freebies under MIT License too.

**Comment by lukeramsden** [comment]
@alexandru-paduraru Ok, let me know what happens so I can start working 😄

**Comment by marcosmoura** [comment]
@alexandru-paduraru Sure! Let's do it. I'm going to release a new version in one week. I think we can start from there. Let me know if you need anything!
And I think will be great to release everything under MIT license! 😄

**Comment by minikN** [comment]
@lukeramsden @alexandru-paduraru @marcosmoura So pumped to see this happening! As Marco said , let me know if you need anything!
